extra-dev/packages/coq-library-undecidability/coq-library-undecidability.dev+8.12/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+version: "dev+8.12"
+maintainer: "forster@ps.uni-saarland.de"
+homepage: "https://github.com/uds-psl/coq-library-undecidability/"
+dev-repo: "git+https://github.com/uds-psl/coq-library-undecidability/"
+bug-reports: "https://github.com/uds-psl/coq-library-undecidability/issues"
+authors: ["Yannick Forster"
+          "Dominique Larchey-Wendling"
+          "Andrej Dudenhefner"
+          "Edith Heiter"
+          "Dominik Kirst"
+          "Fabian Kunze"
+          "Gert Smolka"
+          "Simon Spies"
+          "Dominik Wehr"
+          "Maximilian Wuttke"]
+license: "CeCILL"
```

Comment:
Please specify the version of CECILL you're using. If it's version 2.1, then this should be `CECILL-2.1`. Same for the release.

Reply:
Fixed!